build/installers/liferay-workspace/liferay-workspace.xml

```diff
@@ -51,6 +51,32 @@
     </componentList>
     <postInstallationActionList>
         <runProgram>
+            <customErrorMessage>You may type a wrong email and password for liferay.com </customErrorMessage>
```

Comment:
Message should say "Could not create DXP download token.  Make sure you type correct email address and password and are connected to the internet."